uint8 MODE_WHEEL     = 0
uint8 MODE_OBSTACLE  = 1

Header  header
bool    running
uint8   mode
int16   laser_id
float32 distance_left
float32 distance_right
float32 distance_front
float32 min_angle
float32 max_angle
